robot/packages/cloud_package_manager.go

```diff
@@ -200,25 +157,25 @@ func (m *cloudManager) Sync(ctx context.Context, packages []config.PackageConfig
 
 		m.logger.Debugf("  Downloading from %s", sanitizeURLForLogs(resp.Package.Url))
 
-		// load package from a http endpoint
-		err = m.loadFile(ctx, resp.Package.Url, p)
+		// download package from a http endpoint
+		err = m.downloadPackage(ctx, resp.Package.Url, p)
 		if err != nil {
 			m.logger.Errorf("Failed downloading package %s:%s from %s, %s", p.Package, p.Version, sanitizeURLForLogs(resp.Package.Url), err)
 			outErr = multierr.Append(outErr, errors.Wrapf(err, "failed downloading package %s:%s from %s",
 				p.Package, p.Version, sanitizeURLForLogs(resp.Package.Url)))
 			continue
 		}
-		err = linkFile(m.localDataPath(p), m.localNamedPath(p))
-		if err != nil {
-			m.logger.Errorf("Failed linking package %s:%s, %s", p.Package, p.Version, err)
-			outErr = multierr.Append(outErr, err)
-			continue
+
+		if p.Type == config.PackageTypeMlModel {
+			outErr = multierr.Append(outErr, m.legacyMLModelSymlinkCreation(p))
+			outErr = multierr.Append(outErr, m.legacyCreateMLModelDotDataDirSymlink(p))
+
 		}
 
 		// add to managed packages
 		newManagedPackages[PackageName(p.Name)] = &managedPackage{thePackage: p, modtime: time.Now()}
 
-		m.logger.Debugf("  Sync complete after %dms", time.Since(start).Milliseconds())
+		m.logger.Debugf("  Sync complete after %v", time.Since(start))
```

Comment:
Happy to revert if people are opposed

This was a QoL improvement that I spotted while waiting for my package to download -- `0h1m22s` was a much easier string to read

Reply:
Fine with it, majority will not run in debug mode anyways.